tests/progs/test_jubjub.py

```diff
@@ -84,24 +80,19 @@ def test_basic_point_functionality():
     async def _prog(context):
         p1 = SharedPoint.from_point(context, TEST_POINTS[0])
         p2 = SharedPoint.from_point(context, TEST_POINTS[1])
-        p3 = await SharedPoint.create(context, context.Share(
-            0), context.Share(1), Jubjub(Jubjub.Field(-2)))
 
-        assert await shared_point_equals(p1, p1)
-        assert not await shared_point_equals(p1, p2)
-        assert not await shared_point_equals(p1, p3)
-
-    await run_test_program(_prog, test_runner)
+        # Different curve
+        p3 = deepcopy(p1)
```

Comment:
A deepcopy would be really slow, why do you need this? Can you replace this with something else?

Reply:
I just replaced it with `copy`, which executes much faster.